triage_party/triageparty_configmap.yaml

```diff
@@ -26,6 +26,11 @@ data:
         - https://github.com/cert-manager/cert-manager
         - https://github.com/cert-manager/website
         - https://github.com/cert-manager/release
+        - https://github.com/cert-manager/istio-csr
+        - https://github.com/cert-manager/approver-policy
+        - https://github.com/cert-manager/trust-manager
+        - https://github.com/cert-manager/csi-driver
+        - https://github.com/cert-manager/csi-driver-spiffe
```

Comment:
Did you consider adding:
 * https://github.com/cert-manager/aws-privateca-issuer
 * https://github.com/cert-manager/openshift-routes
 * https://github.com/cert-manager/cert-manager-olm
 * https://github.com/cert-manager/webhook-lib
 * https://github.com/cert-manager/csi-lib
 * https://github.com/cert-manager/sample-external-issuer

Reply:
> https://github.com/cert-manager/aws-privateca-issuer

This one is worked on by AWS folks not us

I was not sure about the other ones. But thinking about it more I think it makes sense to add them- that will ensure that we don't miss issues against those projects